test/e2e/auth/certificates.go

```diff
@@ -323,10 +333,14 @@ var _ = SIGDescribe("Certificates API [Privileged:ClusterAdmin]", func() {
 		for sawAnnotations := false; !sawAnnotations; {
 			select {
 			case evt, ok := <-csrWatch.ResultChan():
-				framework.ExpectEqual(ok, true, "watch channel should not close")
+				if !ok {
+					framework.Failf("watch channel should not close")
```

Comment:
Side question: would it make sense to add a `framework.Fail` which doesn't expand the format string?

This here works because the string contains no `%`. I just always feel uncomfortable about it because it looks wrong.

Reply:
You're right, it would be better to use `framework.Fail`. Sorry about that. It's updated now.

Reply:
Oh, there is a `framework.Fail` and it was added by... me. I completely forgot about that and thought that it was still missing :grin: